mbq/metrics/contrib/django/middleware/timing.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, get_response=None):
     def process_request(self, request):
         if request.path.strip('/') in SETTINGS['EXCLUDED_PATHS']:
             return request
-        setattr(request, '_mbq_metrics_start_time', time())
+        setattr(request, '_mbq_metrics_start_time', monotonic())
```

Comment:
Total nit: probably easier for the uninitiated to read if you use it as `time.monotonic()`

Reply:
+1 for all uses.